packages/orchestration/src/utils/start-helper.js

```diff
@@ -71,7 +72,7 @@ export const provideOrchestration = (
 
   const vowTools = prepareVowTools(zones.vows);
 
-  const chainHub = makeChainHub(agoricNames, vowTools);
+  const chainHub = makeChainHub(zones.chainHub, agoricNames, vowTools);
```

Comment:
Fitrst, just to make sure I understand:

The chainHubs made in start-stakeAtom.js and start-stakeOsmo.js are both made in the heap zone, whereas the chainHub made on this line is durable because it is made in a durable zone. Yes?

If so, why the difference?

Reply:
heapzone is incompatible with durable retry, so all `makeChainHub` will need to take a durable zone. they could keep their maps in heap but there's no harm in keeping them durably.